test/cnvkit-tools/genome2access.cwl

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: "cwl:draft-3.dev2"
```

Comment:
Draft-3.dev2 is the second development version of the draft-3 spec. You should upgrade this to the final version of the draft-3 spec.